utils/convertors/convertors.go

```diff
@@ -24,8 +24,13 @@ import (
 
 // StrToBytes converts string of bytes to integer.
 func StrToBytes(s string) int64 {
+	multiplier := int64(1)
+	if strings.HasSuffix(s, "G") {
```

Comment:
Kubernetes supports "Gi" suffix, too. If we make sure we use "G" always, it's fine.